_config.yml

```diff
@@ -54,6 +54,7 @@ asciidoctor:
   template_dir: site/docs/src/templates/haml
   base_dir: site
   attributes:
+    allow-uri-read: ''
```

Comment:
What does this do? Looks like something to do with embedded images?

Reply:
This allows us to read from the remote github repo via an include, like we do here: https://github.com/FusionAuth/fusionauth-site/pull/854/files#diff-d4cf2c9834026591add8857a4943d0ea8958ed16d31bd84392458a828f5e0587R207